encoderpy/target_encoder.py

```diff
@@ -31,7 +31,7 @@ def target_encoder(X_train, y, cat_columns, X_test = None, prior = 0.5, objectiv
         train_processed : pd.DataFrame
                 The training set, with the categorical columns specified by the argument cat_columns
                 replaced by their encodings.
-        test_processed : pd.DataFrame
+        test_processed : pd.DataFrame (optional)
                 The test set, with the categorical columns specified by the argument cat_columns
                 replaced by the learned encodings from the training set.
```

Comment:
Just for consistency:

```suggestion
        test_processed : pd.DataFrame 
                The test set, with the categorical columns specified by the argument cat_columns
                replaced by the learned encodings from the training set. This is an optional argument.
```